experimental/packages/otlp-grpc-exporter-base/src/configuration/otlp-grpc-env-configuration.ts

```diff
@@ -99,12 +101,15 @@ function getUrlFromEnv(signalIdentifier: string) {
   // - http://example.test:4317 -> use insecure credentials if nothing else is provided
   // - https://example.test:4317 -> use secure credentials from environment (or provided via code)
 
-  const specificEndpoint =
-    process.env[`OTEL_EXPORTER_OTLP_${signalIdentifier}_ENDPOINT`]?.trim();
-  const nonSpecificEndpoint =
-    process.env[`OTEL_EXPORTER_OTLP_ENDPOINT`]?.trim();
+  const specificEndpoint = getStringFromEnv(
+    `OTEL_EXPORTER_OTLP_${signalIdentifier}_ENDPOINT`
+  );
+  const nonSpecificEndpoint = getStringFromEnv(`OTEL_EXPORTER_OTLP_ENDPOINT`);
 
-  return fallbackIfNullishOrBlank(specificEndpoint, nonSpecificEndpoint);
+  return fallbackIfNullishOrBlank(
+    specificEndpoint?.trim(),
+    nonSpecificEndpoint?.trim()
+  );
```

Comment:
I think this whole block could be simplified to (untested, and needs `npm run lint:fix` for formatting):

```js
return getStringFromEnv(`OTEL_EXPORTER_OTLP_${signalIdentifier}_ENDPOINT`) ?? getStringFromEnv('OTEL_EXPORTER_OTLP_ENDPOINT');
```

i.e. `.trim()` and `fallbackIfNullishOrBlank` should no longer be necessary.